### PR TITLE
Orchestration and Scheduling are Declarative

### PR DESCRIPTION
**Pull request by dataders**
## Description & motivation

When this viewpoint was first written, many facets were aspirational rather than descriptive of the current reality. A great example of the is the opinion that analytic code should be version-controlled, which was far from accepted practice five years ago.

In the time since this viewpoint was last edited, there is new dogma amongst data practitioners.

mature analytics projects separate the data plane from the control plane, and minimize the complexity of the control plane.

I’d appreciate any feedback as well as other ideas on how we might update this viewpoint to be more forward looking.

### CODE REVIEW

File: website/docs/community/resources/viewpoint.md [hunk 1]
```diff
@@ -68,3 +68,11 @@ Here’s one example of an automated workflow:
 - code is deployed.
 
 Workflows like this should be built to execute with a single command.
+
+## The orchestration and control flow of analytics pipelines are configured declaratively 
+
+A data professional should maximize the time spent working with data. The configuration of where, how frequently, and in what order the transformations are executed should therefore be declartive, in order to minimize time operationalizing and maintaining workflows.
+
+Pipelines should be deterministic, and not execute different set of transformstions based upsteam conditionals.
```
Comment (runleonarun) [inline website/docs/community/resources/viewpoint.md]:
```suggestion
Pipelines should be deterministic, and not execute different set of transformations based upstream conditionals.
```

File: website/docs/community/resources/viewpoint.md [hunk 2]
```diff
@@ -68,3 +68,11 @@ Here’s one example of an automated workflow:
 - code is deployed.
 
 Workflows like this should be built to execute with a single command.
+
+## The orchestration and control flow of analytics pipelines are configured declaratively
```
Comment (runleonarun) [inline website/docs/community/resources/viewpoint.md:72]:
```suggestion
## Configure the orchestration and control flow of analytics pipelines declaratively 
```

File: website/docs/community/resources/viewpoint.md [hunk 3]
```diff
@@ -68,3 +68,11 @@ Here’s one example of an automated workflow:
 - code is deployed.
 
 Workflows like this should be built to execute with a single command.
+
+## The orchestration and control flow of analytics pipelines are configured declaratively 
+
+A data professional should maximize the time spent working with data. The configuration of where, how frequently, and in what order the transformations are executed should therefore be declartive, in order to minimize time operationalizing and maintaining workflows.
```
Comment (runleonarun) [inline website/docs/community/resources/viewpoint.md:74]: I'm not sure I'm capturing the intent appropriately so definitely open to changes, but here's my suggestion to make this sentence active:
```suggestion
A data professional should maximize their time spent working with data by minimizing time operationalizing and maintaining workflows. With this efficiency in mind, you should declaratively configure where, how frequently, and in what order you execute transformations.
```

**Review by runleonarun — COMMENTED** [review]
@dataders just a first pass at the language!

**Comment by nghi-ly** [comment]
Chatted with dataders regarding status of this. These changes will need a wider audience to review and sign off. Can close this for now.

**Pull request closed without merging**